### PR TITLE
Assingment

### PR DESCRIPTION
**Pull request by Shravani1510**

### CODE REVIEW

**Comment by floe** [comment]
Your write method just creates an empty file (-1), which is therefore not a valid PPM (-1). I also need the full name for each team member.

**Comment by Shravani1510** [comment]
Shravani kumari pariseka (119483)
Nandini maheswari perugu ( 119466)

File: Image.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+			data[pos+1] = (byte)g;
+			data[pos+2] = (byte)b;
+			
+			
+		}
+public void write(String filename) throws FileNotFoundException {
+	// TODO Auto-generated method stub
+	FileOutputStream fos = new FileOutputStream(filename);
+	 try {
+
```
[the remark this answers is not in the document]
Comment (Shravani1510) [reply Image.java]: okay.
Can I correct  my mistake now.

[the remark this answers is not in the document]
Comment (Shravani1510) [reply Image.java]: Okay,
Thank you.

**Pull request closed without merging**